examples/console/go.mod

```diff
@@ -11,3 +11,22 @@ require (
 	github.com/stellar/go v0.0.0-00010101000000-000000000000
 	gopkg.in/yaml.v2 v2.3.0 // indirect
 )
+
+require (
```

Comment:
same question here. These didn't show up for me locally when I upgraded to 1.17

Reply:
They won't show up until the `go` directive at the top of the file is changed to `1.17`. `go mod tidy` respects the go directive to some degree, so even if you're using go 1.17, if the file says 1.16, it uses the logic from go 1.16 for how to generate the file, but nothing else.